### ISSUE TITLE
Making the App Responsive and Look Awesome on Mobile Devices

### ISSUE THREAD

**Issue by kenmazaika** [stevenlecreates/Splurty#1]
On mobile things don't look as good as they could be.

One thing you could look into is to add the "mobile meta tag" to your app, so the responsive effect of Bootstrap actually shows up on Mobile.

Let me know in the comments once you figured that step out :)

**Comment by stevenlecreates** [comment]
Hey Ken, 

So I've tried adding this: (Which I googled and found on bootstraps page)

<meta name="viewport" content="width=device-width, initial-scale=1">

to the <head> section of the application.html.erb file. 

I've tested it out by resizing my browser screen and viewing the application on my phone in both landscape and portrait modes. I'm not sure if it's actually made a difference though.

Was that the mobile meta tag you had in mind and is it placed in the correct place for it to take effect?

Most recent code has been pushed to github!

Thanks,
Steven
